packages/retail-ui/components/TokenInput/TokenInput.tsx

```diff
@@ -338,7 +344,8 @@ export default class TokenInput<T = string> extends React.PureComponent<TokenInp
     const tokens = this.state.activeTokens
       .map(token => this.props.selectedItems.indexOf(token))
       .sort()
-      .map(index => this.props.selectedItems[index]);
+      .map(index => this.props.selectedItems[index])
+      .map(this.props.valueToString);
```

Comment:
Я бы обезопасил вызов и сделал:
```suggestion
      .map(item => this.props.valueToString(item));
```
Иначе по типу функция принимает один аргумент, а на деле в неё прилетают 3